### PR TITLE
13 fix table pagination bug

### LINKED ISSUE #13: Fix table pagination bug
- [ ] table pagination is not working
- [ ] row selection is not working
- [ ] table overflow

### PR DESCRIPTION
**Pull request by abhishekt98**
Fixes #13

### CODE REVIEW

File: app/javascript/src/data/index.js [hunk 1]
```diff
@@ -6,14 +6,20 @@ const notes = Array(4).fill({
...
+    id: i,
+    person: {
+      name: "Ronald Richards",
+      role: "owner",
+    },
+    email: "albert@borer.com",
+    createdDate: "Feb, 5, 2021",
+  };
+  contacts.push(obj);
+}
```
Comment (karthiknmenon) [inline app/javascript/src/data/index.js:21]: Why do we need to do it this way?

File: app/javascript/src/components/notes/index.jsx [hunk 2]
```diff
@@ -13,6 +14,24 @@ const Index = () => {
   const notes = data.notes;
   const [showDeleteAlert, setShowDeleteAlert] = useState(false);
   const [showPane, setShowPane] = useState(false);
+  const [loading, setLoading] = useState(true);
```
Comment (karthiknmenon) [inline app/javascript/src/components/notes/index.jsx:17]: `const [isLoading, setIsLoading] = useState(true);`

File: app/javascript/src/components/contacts/index.jsx [hunk 3]
```diff
@@ -13,12 +14,29 @@ const Index = () => {
   const contacts = data.contacts;
   const [showPane, setShowPane] = useState(false);
   const [showDeleteAlert, setShowDeleteAlert] = useState(false);
+  const [loading, setLoading] = useState(true);
```
Comment (karthiknmenon) [inline app/javascript/src/components/contacts/index.jsx:17]: const [isLoading, setIsLoading] = useState(true);

**Review by karthiknmenon — COMMENTED** [review]
Please refrain from adding branch numbers in PR titles. Please use proper titles.

File: app/javascript/src/data/index.js (hunk 1 above)
Comment (abhishekt98) [reply app/javascript/src/data/index.js:21]: Is this implementation is okay ?
```
const CONTACTS_DATA = Array(50)
  .fill(null)
  .map((ele, index) => ({
    id: index,
    person: {
      name: "Ronald Richards",
      role: "owner",
    },
    email: "albert@borer.com",
    createdDate: "Feb, 5, 2021",
  }));
```

**Pull request merged**